test/commands/source/open.test.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { $$, expect, test } from '@salesforce/command/lib/test';
```

Comment:
I'd rather the source plugin not use this method of testing because it seems to have interactions with other stubs and spies.  If you look at the other command tests in the plugin there is a pattern to follow that I think works better.